app/views/partials/personal_key/_key.html.erb

```diff
@@ -4,10 +4,10 @@
   </h2>
   <div class="margin-top-6 margin-bottom-6 padding-x-0 tablet:padding-x-1 padding-y-2 center bg-white border-width-2px separator-text bg-pk-box">
     <%- code.split('-').each do |word|
-      -%><div class="inline fs-20p bold navy"
-        ><code class="monospace" data-personal-key="word"
+      -%><div class="inline"
+        ><strong><code class="separator-text__code monospace" data-personal-key="word"
           ><%= word
-        %></code
+        %></strong></code
       ></div
```

Comment:
👀 personal key... looks like these changes *should not* affect whitespace, spacing, but that's something that could affect copy-paste which might affect verification

Reply:
> something that could affect copy-paste which might affect verification

Good lead, though from what I can tell, the actual copied text is defined elsewhere:

https://github.com/18F/identity-idp/blob/f69733afbd5b00f72c3598d05ddbcd00e67393ff/app/views/shared/_personal_key.html.erb#L19-L20

https://github.com/18F/identity-idp/blob/fc1ec7738b580e4567bfae038cb0f5b50ece0d5b/app/javascript/packs/clipboard.js#L1-L7